src/org/jstrava/connector/JStravaV3.java

```diff
@@ -283,12 +283,16 @@ public Activity updateActivity(int activityId, HashMap optionalParameters) {
 
     @Override
     public List<Activity> getCurrentAthleteActivities() {
```

Comment:
This changes the behavior of  getCurrentAthleteActivities() and potentially breaks existing implementations by returning 100s of results instead of <=30. 

This has implications for network access times, API usage among other things. 

Recommend not merging and create a new method instead.